src/main/java/net/mcreator/minecraft/ElementUtil.java

```diff
@@ -169,6 +169,12 @@
 		return retval;
 	}
 
+	public static List<DataListEntry> loadAllVillagerProfessions(Workspace workspace) {
+		List<DataListEntry> retval = getCustomElementsOfType(workspace, BaseType.VILLAGER);
```

Comment:
At the moment villager trade elements use `VILLAGER` as base type, so they appear in the villager profession dropdowns even when they shouldn't

Reply:
Fixed! I changed VillagerTrades's BaseType. The `VILLAGER` base type will be used by the future Villager mod element that will allow people to create their own Villager Profession, and the custom professions will appear in the list.